command/license.go

```diff
@@ -0,0 +1,112 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/mitchellh/cli"
+)
+
+var _ cli.Command = &LicenseCommand{}
+
+type LicenseCommand struct {
+	Meta
+}
+
+func (l *LicenseCommand) Help() string {
+	helpText := `
+Usage: nomad license <subcommand> [options] [args]
+	
+This command has subcommands for managing the Nomad Enterprise license.
+For more detailed examples see:
+https://www.nomadproject.io/docs/commands/license/
+
+Install a new license from a file:
+	$ nomad license put @nomad.license
+
```

Comment:
Update the docs to only accept the file path or `-`?

Reply:
woops thanks